monitoring/grafana/jsonnet/panels/compactor.libsonnet

```
@@ -0,0 +1,163 @@
+    |||;
+
+    timeseriesPanel.new(
+      title='Compaction Success & Failure Counts (Detailed)',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ pod }} - group{{ group }} - {{ result }}',
+      )
+    ) + span.span('half'),
+
+<<<<<<< HEAD
+    max_compaction_task_duration_overview::
+=======
+  max_compaction_task_duration::
+>>>>>>> 86d6f4b (style: format the jsonnet files)
+    local expr = |||
+      histogram_quantile(1, sum by (le) (rate(state_store_compact_task_duration_bucket{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])))
+    |||;
+
+    timeseriesPanel.new(
+      title='Longest Compaction Duration (Overview)',
+      unit='s',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='overview',
+      )
+    ) + span.span('half'),
+
+<<<<<<< HEAD
+    max_compaction_task_duration_detailed::
+=======
+  median_compaction_task_duration::
+>>>>>>> 86d6f4b (style: format the jsonnet files)
+    local expr = |||
+      histogram_quantile(1, sum by (job, le) (rate(state_store_compact_task_duration_bucket{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])))
+    |||;
+
+    timeseriesPanel.new(
+      title='Longest Compaction Duration (Detailed)',
+      unit='s',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ pod }}',
+      )
+    ) + span.span('half'),
+
+    median_compaction_task_duration_overview::
+    local expr = |||
+      histogram_quantile(0.5, sum by (le) (rate(state_store_compact_task_duration_bucket{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])))
+    |||;
+
+    timeseriesPanel.new(
+      title='Median Compaction Duration (Overview)',
+      unit='s',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='overview',
+      )
+    ) + span.span('half'),
+
+    median_compaction_task_duration_detailed::
+    local expr = |||
+      histogram_quantile(0.5, sum by (le, pod) (rate(state_store_compact_task_duration_bucket{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])))
+    |||;
+
+    timeseriesPanel.new(
+      title='Median Compaction Duration (Detailed)',
+      unit='s',
+    ).addTarget(
+      prometheus.target(
+        expr=strings.trim(expr),
+        legendFormat='{{ pod }}',
+      )
+    ) + span.span('half'),
+
+  compaction_read_write_rate::
+    local expr1 = |||
+      sum(rate(storage_level_compact_write{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])) by(pod)
+    |||;
+    local expr2 = |||
+      sum(rate(storage_level_compact_read_next{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])) by(pod) + sum(rate(storage_level_compact_read_curr{namespace="$namespace", risingwave_name="$instance"}[$__rate_interval])) by(pod)
```

Format the query to make it more human-readable.
```suggestion
      sum by (pod) (
        rate(storage_level_compact_read_next{namespace="$namespace",risingwave_name="$instance"}[$__rate_interval])
      )
    +
      sum by (pod) (
        rate(storage_level_compact_read_curr{namespace="$namespace",risingwave_name="$instance"}[$__rate_interval])
      )
```